contributing-tips.rst

```diff
@@ -0,0 +1,174 @@
+
+An issue might take a long time because:
+
+- it is complex and requires lots of hours 
+- you only have a short period of time each day to work on it
+- you are new to the project and are having to learn as you go
+
+The thing that matters is: **Are you actively working on the issue, and making progress, at least a little bit?**
+
+If you are actively working on it, we do not want someone else to jump on and try to work on it at the same time. So please keep the community informed of your work by filing a ``[WIP]`` (work in progress) PR and comitting to it as you work.
+
+.. _issues-only:
+
+Issues only
+----------------
+
+All PRs must be directly connected to open issues. PRs should not represent suggestions, good ideas, or independent intiative. 
+
+If you have a good idea, file an issue. If you are curious about whether something should be an issue, chat with one of the core team in the `#docs-code` channel on the `Slack <https://opendatakit.slack.com>`_.
+
+Once you have filed an issue, wait for comment and approval before diving into the work. We do not want surprise PRs.
+
+.. _use-odk:
+
+Actually install and use Open Data Kit or other tools
+----------------------------------------------------------
+
+You cannot write effectively about tools you have not used. If you're going to write or edit documentation about any of the apps in the ODK ecosystem, you need to spend some time actually using it. 
+
+Before diving into writing documentation, follow the :doc:`getting-started` guide so that you are familiar with the core ODK tools.
+
+This is also true of writing about Sphinx or any of our documentation build tools. Reading existing documentation is not enough to write about something.
+
+.. _do-the-thing:
+
+And actually do the thing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are writing about a specific process (installing an application, for example), you need to actually complete the process yourself. If possible, follow your own instructions *after* writing them to make sure they make sense.
+
+.. _always-build-locally:
+
+Always build locally
+----------------------
+
+Before submitting a PR, run the build locally to make sure you do not produce any errors or warnings. **We do not accept PRs that produce errors or warnings.**
+
+It is best to run the build frequently as you work. You'll often catch simple mistakes that are harder to track down later.
+
+.. _no-imposters:
+
+You are not an imposter
+--------------------------
+
+`Imposter syndrome <https://en.wikipedia.org/wiki/Impostor_syndrome>`_ is the feeling that you are not good enough or accomplished enough to do the work you are doing.
+
+We all feel this way sometimes, and that's okay. But it is important to realize that **you are not an imposter.** 
+
+You can contribute to this community, no matter your background or skills. 
+
+- If there is something you don't know how to do, you can ask.  
```

Comment:
ask either on the issue, slack channel or the forum
Forum can also be mentioned as a source to clear doubts and get different ideas.